pkg/models/app.go

```diff
@@ -7,7 +7,7 @@ type App struct {
 	AppID                 string     `json:"appId"`
 	AppName               string     `json:"appName"`
 	NumOfDeployedVersions *int       `json:"numOfDeployedVersions,omitempty"`
-	NumOfCurrentInstalls  *int       `json:"numOfCurrentInstalls"`
-	NumOfAppLaunches      *int       `json:"numOfAppLaunches"`
+	NumOfCurrentInstalls  *int       `json:"numOfCurrentInstalls,omitempty"`
+	NumOfAppLaunches      *int       `json:"numOfAppLaunches,omitempty"`
```

Comment:
It is missing the swagger.yaml. Just run the make setup locally and commit the file.

Reply:
:+1: Good spot

Reply:
@camilamacedo86 make setup didn't update the swagger.yaml. maybe it ignores this change. 

```bash
make setup
Installing errcheck dependence:
go get -u github.com/kisielk/errcheck
Setting up Git hooks:
ln -sf $PWD/.githooks/* $PWD/.git/hooks/
Installing application dependencies:
dep ensure
make build_swagger_api
make[1]: Entering directory '/home/austincunningham/gostuff/src/github.com/aerogear/mobile-security-service'
Installing Swagger dep:
go get -u github.com/go-swagger/go-swagger/cmd/swagger
Updating Swagger api:
cd cmd/mobile-security-service; swagger generate spec -m -o ../../api/swagger.yaml
make[1]: Leaving directory '/home/austincunningham/gostuff/src/github.com/aerogear/mobile-security-service'
```
git didn't pick up any changes in `swagger.yaml`

Reply:
just looking at swagger, this change only ignores when null , zero is not null so looks like the swagger is fine.